packages/augur-ui/src/modules/reporting/common.tsx

```diff
@@ -493,6 +511,13 @@ export class DisputingBondsView extends Component<
     const remaining = convertAttoValueToDisplayValue(
       createBigNumber(stakeRemaining)
     );
+
+    const gasEstimate = formatGasCostToEther(
```

Comment:
Does this need to be in the render method. It doesnt relay on any user inputs

Reply:
Nope, I can hoist this up and pass it in as a prop